### ISSUE TITLE
OAuth 2.0 Access Token Size

### ISSUE THREAD

**Issue by SvenSchoene** [googleapis/google-api-php-client#2079]
This morning we received this email:

> Google Cloud Platform will roll out security and reliability improvements that will increase the sizes of OAuth 2.0 tokens. Please follow the instructions below to use the documented token size limits of 2048 bytes and to remove any logic in your services/code that restricts its ability to process access tokens of certain sizes before August 23, 2021.
>
> Hello Google Cloud Customer,
>
> We’re writing to let you know that on August 23, 2021, we will roll out security and reliability improvements that will increase the sizes of OAuth 2.0 access tokens that your services may be using to call Google Cloud Platform APIs.
>
> The access token size increase will only impact services that don’t conform to the access token size limits of 2048 bytes established on Google’s Developer guide and RFC 6749. If your service falls into this category, please continue reading and take action as outlined below.

1. Does this apply to this package? 
2. Is this increase of the token size an issue with the current version?
3. Do you know if it ever was an issue with a previous version? We have older servers deployed and I wonder whether they need to be upgraded. I couldn't find anything in the source code to indicate that.

**Comment by yannickl88** [comment]
After closer inspection, the error came from a different system and was nog related to google. So my previous comment can be discarded.

**Comment by bshaffer** [comment]
I do not believe this will have any effect here, as we do not restrict token sizes.

**Comment by SvenSchoene** [comment]
I see. So if the token size increases, but this package does not check for / restrict the token size, then I too would assume that everything would keep working as it should.

**Comment by bshaffer** [comment]
That is my understanding, yes
